### PR TITLE
User story#9

### PR DESCRIPTION
**Pull request by VinayChavhan**
Added happiness service

### CODE REVIEW

**Review by VidhinASU — CHANGES_REQUESTED** [review]

**Comment by skuma251** [comment]
Please make sure you are not raising the PR directly for main, it should go to develop first and tested their, only then it can go to main

File: src/happinessservice/index.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+
+
+
+// Endpoint for adding bv as a custom field
+app.post("/AddHappiness", async (req, res) => {
+  const { username, password, projectId, name, description } = req.body
+  let { token } = req.body
+  if (!token) {
+    token = await getToken(username, password)
```
Comment (VidhinASU) [inline src/happinessservice/index.js:17]: The function getToken is not defined

**Review by VinayChavhan — COMMENTED** [review]
made all suggested changes, please review

**Review by VidhinASU — APPROVED** [review]

**Review by shivanjay-wagh — APPROVED** [review]
looks good

**Pull request merged**